.github/workflows/nightly.yaml

```diff
@@ -25,6 +31,14 @@ jobs:
       - run: mvn -version
       - name: Unit Tests
         run: mvn install --errors --batch-mode --no-transfer-progress -Dcheckstyle.skip -T 1C
+      - name: Create issue
```

Comment:
My guess: the execution does not reach "Create` issue" step if the "mvn install ..." above fails.